src/block/map/save.js

```diff
@@ -90,7 +91,14 @@ export const Save = props => {
 					<div
 						data-map-options={ JSON.stringify( mapOptions ) }
 						data-marker-options={ JSON.stringify( markerOptions ) }
-						data-icon-options={ JSON.stringify( getIconOptions( attributes ) ) }
+						data-icon-options={ JSON.stringify(
+							applyFilters(
+								'stackable.map.util.applySVGAttributes',
```

Comment:
I recommend changing the name of this filter because if you look at where this filter is being used, it's not related to `util.applySVGAttributes`. I feel like `stackable.map.icon-options` makes more sense here:

```suggestion
								'stackable.map.icon-options',
```

Also change the corresponding code that uses this filter